install_config/aggregate_logging_sizing.adoc

```diff
@@ -192,6 +192,18 @@ journal file corruption.
 link:https://rhn.redhat.com/errata/RHBA-2016-1026.html[This issue is resolved in
 later versions of systemd].
 
+*Fluentd*
+
+As of {product-title} 3.7, OpenShift tests its logging architecture with 1KB log
+messages, approximately 1000 messages/second. This is the upper limit
```

Comment:
messages per second

Reply:
Can we strike the 1,000 messages/second, and only report 850 messages/second?

Why are reporting "unstable" numbers, as implied by line 201?

Reply:
Ok - how about this:
```
As of {product-title} 3.7, with 1 kilobyte sized log messages, the logging architecture tests have achieved a stable logging rate of 800-900 messages per second per node.
```

Reply:
Sounds good, except isn't 800-900 messages a second for a specific configuration?  Would faster CPUs, networking, etc. impact this?

If so, we should report what the configuration was where we saw these numbers.